src/database/schema.sql

```diff
@@ -123,11 +123,12 @@ CREATE TABLE ProfileSkills (
 CREATE TABLE Organisations (
```

Comment:
For `Organisations`, additionally add 
`locations VARCHAR`, optional field
`story TEXT` optional field

Reply:
Do add these columns on the create and update endpoints